### PR TITLE
feat: add package 'string-machine'

### PR DESCRIPTION
**Pull request by SpotlightKid**

### CODE REVIEW

**Review by cbix — CHANGES_REQUESTED** [review]
Didn't test VST, but the LV2 works great :) Please check my comments!

File: packages/string-machine/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+groups=(pro-audio lv2-plugins vst-plugins)
+source=("https://github.com/jpcima/string-machine/releases/download/v$pkgver/$pkgname-$pkgver.tar.gz"
+        'string-machine-dpf-makefile.diff')
+sha256sums=('5b0d2eb2185199c1de6c7c35700a2caafbc6ba564529a2e7614c15c3aceacc6f'
+            '87d3c13a567ab156b4cc44952b27ed12400f25f5099556d667595f7d2b2492d7')
+
+prepare() {
+  cd "$pkgname-$pkgver"
+  # Update DPF base makefile to
+  # https://github.com/DISTRHO/DPF/blob/05d91f5852f4bccfd2bce1d4d2e2b3036e29db03/Makefile.base.mk
```
[the remark this answers is not in the document]
Comment (SpotlightKid) [reply packages/string-machine/PKGBUILD]: Done in d4aef21.

File: packages/string-machine/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+pkgname=string-machine
+_plugin_uri="http://jpcima.sdf1.org/lv2/$pkgname"
+pkgver=0.1.0
+pkgrel=1
+pkgdesc="A virtual-analog string ensemble synthesizer LV2 and VST2 plugin"
+arch=(x86_64)
```
[the remark this answers is not in the document]
Comment (SpotlightKid) [reply packages/string-machine/PKGBUILD]: Done in d4aef21.

**Review by cbix — APPROVED** [review]
Awesome, thanks 🎉

**Pull request merged**